en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md

```diff
@@ -0,0 +1,237 @@
+# Email Connector Example
+
+Email Connector can be used to perform operations using standard protocols such as SMTP, IMAP and POP3. 
+
+## What you'll build
+
+This example explains how to use Email Connector to send an email and retrieve the same email from GMail. The user sends a payload with the recipients, content and attachments of the email. Then, by invoking another API resource, the content of the sent email will be retrieved. 
+
+It will have two HTTP API resources, which are `send` and `retrieve`. 
```

Comment:
The example consists of an API named as `EmailConnector API` with two resources `send` and `retrieve`.

Reply:
Fixed